go_sequencer/database/utils.go

```diff
@@ -99,10 +98,6 @@ func InitSQLDB(port int, host, user, password, name string) (*sqlx.DB, error) {
 
 // InitTestSQLDB opens test PostgreSQL database
 func InitTestSQLDB() (*sqlx.DB, error) {
-	err := godotenv.Load("../.env")
-	if err != nil {
-		panic("Error loading .env file")
-	}
```

Comment:
Loading env file is necessary to test locally.

Reply:
Good point @kaiden-araki. The reason I removed it is because `task` provides a convenient way of injecting the env file contents so that we can `os.Getenv("VAR_NAME")` without loading the env file in the code. You can give it a try after installing task with `brew` and run `task test-historydb`.

Reply:
@kaiden-araki I restored loading `.env` in `utils.go` but changed `panic()` to `log.Info()` since github actions doesn't load the file. Please, let me know if you have any alternative suggestions.

Reply:
@Intizar-T Thanks, it looks good to me.